cla_frontend/assets-src/javascripts/app/js/controllers/call_centre/eod_details_modal.js

```diff
@@ -0,0 +1,118 @@
+(function() {
+  'use strict';
+
+  angular.module('cla.controllers.operator')
+    .controller('EODDetailsModalCtrl',
+      ['$scope', 'case', 'eod_details', 'tplVars', '$modalInstance', 'EXPRESSIONS_OF_DISSATISFACTION', 'EXPRESSIONS_OF_DISSATISFACTION_FLAGS', '$q', '$timeout', 'flash', 'postal',
+        function($scope, $case, eod_details, tplVars, $modalInstance, EXPRESSIONS_OF_DISSATISFACTION, EXPRESSIONS_OF_DISSATISFACTION_FLAGS, $q, $timeout, flash, postal) {
+
+          // template vars
+          tplVars = angular.extend({
```

Comment:
The text seems to be unchanged throughout the flow. Should it go directly to template?